include/velocypack/Slice.h

```diff
@@ -748,133 +794,40 @@ class Slice {
     VELOCYPACK_ASSERT(h >= 0xc0 && h <= 0xc7);
 
     std::vector<uint8_t> out;
-    ValueLength length = readIntegerNonEmpty<ValueLength>(_start + 1, h - 0xbf);
+    ValueLength length = readIntegerNonEmpty<ValueLength>(start() + 1, h - 0xbf);
     checkOverflow(length);
     out.reserve(static_cast<std::size_t>(length));
-    out.insert(out.end(), _start + 1 + h - 0xbf,
-               _start + 1 + h - 0xbf + length);
+    out.insert(out.end(), start() + 1 + h - 0xbf,
+               start() + 1 + h - 0xbf + length);
     return out;
   }
 
-  // get the total byte size for the slice, including the head byte
+  // get the total byte size for the slice, including the head byte, excluding tags
   ValueLength byteSize() const {
-    auto const h = head();
```

Comment:
IMHO this change should be reverted, because it changes the behavior for all embedders and will cause subtly out-of-bounds errors for applications that want to use tagged values.
Of course a case for `Tagged` needs to be added to the old implementation to accommodate for tagged values.